src/routes/+page.svelte

```diff
@@ -1,2 +1,203 @@
-<h1>Welcome to SvelteKit</h1>
-<p>Visit <a href="https://svelte.dev/docs/kit">svelte.dev/docs/kit</a> to read the documentation</p>
+<script>
+  export let form;
+  let copied = false;
+  let url = '';
+
+  function handleSubmit() {
+    
+  }
+
+  async function copyToClipboard(text) {
+    try {
+      await navigator.clipboard.writeText(text);
+      copied = true;
+      setTimeout(() => copied = false, 2000);
+    } catch (err) {
+      console.error('Failed to copy:', err);
+    }
+  }
+
+  $: if (form && form.longUrl) {
```

Comment:
svelte 5 does not support that, please use derived (and anyway better form?.longUrl